baseband/vdif/header.py

```diff
@@ -28,6 +32,41 @@
                   precision=9)
 
 
+_vdif_edv_header_classes = {}
```

Comment:
Done, though this breaks back-compatibility for anyone (i.e. almost certainly no one) who's manually appending to that dict.